docs/components/pages/pack-home/PackBenchmarksGraph.tsx

```diff
@@ -238,11 +250,44 @@ const GraphTimer = ({ turbo, timer }: { turbo: boolean; timer: number }) => {
           />
         </div>
       )}
-      <p className="font-mono">{timer.toFixed(2)}s</p>
```

Comment:
In case anybody is wondering, this `toFixed(N)` has all sorts of IEEE754-induced weirdness.

In particular:
```
console.log(0.015.toFixed(2), 0.087.toFixed(2))
// => 0.01 0.09
```